ruby track/exercise16.rb

```diff
@@ -0,0 +1,63 @@
+# Exercise: 16 Highlight Search Result
+# Write a search method that searches for a letter or word in a user input string and highlights the searched element. Also it should display the total no of searches found. Eg: If user enters string to be searched as "Can you can a can as a canner can can a can?" and word to be searched is 'can' then output should be: 
+# (Can) you (can) a (can) as a (can)ner (can) (can) a (can)? Total Occurences found: 7
+# Sample inputs
+# [name]
+# 1
+# [/name]
+# [input]
+# [/input]
+# [output]
+# Please provide an input
+# [/output]
+
+# ===
+
+# [name]
+# 2
+# [/name]
+# [input]
+# "Can you can a can as a canner can can a can?" "can"
+# [/input]
+# [output]
+# (Can) you (can) a (can) as a (can)ner (can) (can) a (can)?
+# Total occurrences found: 7 
+# [/output]
+
+# ===
+
+# [name]
+# 3
+# [/name]
+# [input]
+# "Can you can a can as a canner can can a can?" "can can"
+# [/input]
+# [output]
+# Can you can a can as a canner (can can) a can?
+# Total occurrences found: 1 
+# [/output]
+
+# ===
+
+# [name]
+# 4
+# [/name]
+# [input]
+# "Good morning everyone" "hello"
+# [/input]
+# [output]
+# Good morning everyone
+# Total occurrences found: 0 
+# [/output]
+
+
+def high(s,s1)
+  str=s.gsub("#{s1}","(#{s1})")
+  s.gsub(/#{s1.capitalize}/, "(#{s1.capitalize})")
```

Comment:
you can chain gsub method. Also you are not storing the result of this line anywhere. This will only return the count and not print the gsub'ed string